config/initializers/scrna_config.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
```

Comment:
Wondering why this file shows in the PR, as I added it a while back.

Reply:
I have copied it from Limber and added some options to the same config as calculations required the same settings.

Reply:
I have now added a comment to specify that the file was copied from Limber and modified with additional entries for the cDNA Prep submission.